formSchemas.js

```diff
@@ -216,7 +216,7 @@ const maritalStatusSchema = {
 }
 
 const addressSchema = {
-  streetName: {
+  line1: {
     isEmpty: {
       errorMessage: 'errors.address.streetName.empty',
```

Comment:
Can you change this to `errors.address.line1.empty` just so it all matches up (and then update the locales files)?